package.xml

```diff
@@ -1,35 +1,31 @@
-<package>
-  <name>yocs_velocity_smoother</name>
+<package format="2">
+  <name>velocity_smoother</name>
   <version>0.12.1</version>
   <description>
      Bound incoming velocity messages according to robot velocity and acceleration limits.
   </description>
   <author>Jorge Santos Simon</author>
   <maintainer email="jihoonl@yujinrobot.com">Jihoon Lee</maintainer>
   <license>BSD</license>
-  <url type="website">http://ros.org/wiki/yocs_velocity_smoother</url>
-  <url type="repository">https://github.com/yujinrobot/yujin_ocs</url>
-  <url type="bugtracker">https://github.com/yujinrobot/yujin_ocs/issues</url>
-  
-  <buildtool_depend>catkin</buildtool_depend>
+  <url type="repository">https://github.com/kobuki-base/velocity_smoother</url>
+  <url type="bugtracker">https://github.com/kobuki-base/velocity_smoother/issues</url>
```

Comment:
I've been pointing the website url at ros index. i.e. for this package: `https://index.ros.org/v/velocity_smoother/github-kobuki-base-velocity_smoother`

Reply:
In this case, though, these URLs are for the repository and bugtracker, respectively.  So I think they make sense where they point to now.  Would you like me to add another URL pointing to the rosindex?

Reply:
Those two are fine. The original had a third url for the website - originally the ROS2 wiki. I've been pointing my repos at the ROS2 index now since that is generally where you can discover most details about the package.

Reply:
Committed in eed3d455